examples/pyolite.ipynb

```diff
@@ -32,7 +32,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from IPython.display import Markdown, HTML, JSON, Latex"
+    "from IPython.display import display, Markdown, HTML, JSON, Latex"
```

Comment:
Do we need to import display? Is it not injected into the user scope automatically?

Reply:
Yeah, I think we need not import it. Let me change it then.